src/commands/play.rs

```diff
@@ -75,40 +79,28 @@ pub async fn _play(
             EnqueueType::URI | EnqueueType::SEARCH => match flag {
                 PlayFlag::PLAYTOP => {
                     let track = queue.get(1).unwrap();
-                    let embed = create_queued_embed("Added to top", track, estimated_time).await;
+                    let embed =
+                        create_queued_embed("⌛  Added to top", track, estimated_time).await;
 
-                    interaction
-                        .edit_original_interaction_response(&ctx.http, |r| {
-                            r.content(" ").add_embed(embed)
-                        })
-                        .await?;
+                    edit_embed_response(&ctx.http, interaction, embed).await?;
                 }
                 PlayFlag::DEFAULT => {
                     let track = queue.last().unwrap();
-                    let embed = create_queued_embed("Added to queue", track, estimated_time).await;
+                    let embed =
+                        create_queued_embed("⌛  Added to queue", track, estimated_time).await;
 
-                    interaction
-                        .edit_original_interaction_response(&ctx.http, |r| {
-                            r.content(" ").add_embed(embed)
-                        })
-                        .await?;
+                    edit_embed_response(&ctx.http, interaction, embed).await?;
                 }
             },
             EnqueueType::PLAYLIST => {
-                interaction
-                    .edit_original_interaction_response(&ctx.http, |response| {
-                        response.content("Added playlist to queue!")
-                    })
-                    .await?;
+                edit_response(&ctx.http, interaction, "⌛  Added {} tracks to queue").await?;
```

Comment:
This is busted! It isn't editing the existing embed and probably missing a `format!`. 

![image](https://user-images.githubusercontent.com/5618493/150650415-9843ebce-ad1e-4fd0-8cc1-f10af0ea9c83.png)

Reconsider ⌛ here as well.